templates/vm.conf.j2

```diff
@@ -21,8 +21,9 @@ vcpus                   = {{ vm.vcpus|default(xen_vman_default_vcpus) }}
 cpus                    = "{{ vm.cpu_str|default(xen_vman_default_cpu_str) }}"
 
 vif                     = [ {% for interface in vm.interfaces -%}
+	{%- set ips = [ interface.ip | default(None), interface.ipv6 | default(None) ] | select('string') -%}
 	'mac={{- interface.mac -}}
-	{%- if interface.ip is defined -%},ip={{- interface.ip|ipaddr('address') -}}{%- endif -%}
```

Comment:
Why don't we need `|ipaddr('address')`anymore?

Reply:
I wondered why we needed it, but it's due to https://github.com/stuvusIT/vm-facts/blob/e1a3399312550f9d3578915058315faff437e541/scripts/hypervisor-facts.py#L88 so I fixed it in my recent commit.